Formula/percona-server.rb

```diff
@@ -8,8 +8,8 @@ class PerconaServer < Formula
   revision 2
 
   livecheck do
-    url "https://www.percona.com/downloads/Percona-Server-LATEST/"
-    regex(/value=.*?Percona-Server[._-]v?(\d+(?:\.\d+)+-\d+)["' >]/i)
+    url "https://docs.percona.com/percona-server/8.0/index.html"
+    regex(/href=.*?v?(\d+(?:\.\d+)+-\d+)\.html/i)
```

Comment:
```suggestion
    url "https://docs.percona.com/percona-server/latest/index.html"
    regex(/href=.*?v?(\d+(?:[.-]\d+)+)\.html/i)
```

* The `8.0` URL would need to be updated for each major/minor release but substituting `latest` redirects to the correct URL without any versioning issue.
* There have been some tags like `5.1.55`, with no numeric suffix, and `5.6.51-93.0`, with a suffix including a dot. Simply using `[.-]` in the repeating part will handle the current format (e.g., `8.0.33-25`) as well as those previous formats, so there's something to be said for using this approach here.